src/Framework/AssemblyLoadBuildEventArgs.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+//
+
+#nullable disable
+
+using System;
+using System.IO;
+using Microsoft.Build.Shared;
+
+namespace Microsoft.Build.Framework
+{
+    public class AssemblyLoadBuildEventArgs : BuildMessageEventArgs
```

Comment:
Given this is a public type, is there a reason you all don't default to `sealed`?

Reply:
We haven't had this rule but we should.